recipes/openfisca-france/bld.bat

```diff
@@ -0,0 +1,4 @@
+ECHO "Renaming rg.exe, see https://github.com/conda-forge/staged-recipes/issues/17519"
+move C:\Miniconda\Library\bin\rg.exe C:\Miniconda\Library\bin\rg-desactivate.exe
```

Comment:
Understand why this is needed, but wonder if there is a way to do this without altering the system Miniconda install. If a user runs this build locally, it may have a surprising effect

cc @jaimergp (in case you have thoughts here)

Reply:
I can think of two workarounds (not necessarily exclusive):

* Guard this under an if check that tests whether we are running on CI or not.
* Undo this in two possible places: if the `build.sh` script fails, or if it succeeds, in the test phase. It's very dirty but it should work.

Reply:
Both of those sound reasonable

Reply:
> Guard this under an if check that tests whether we are running on CI or not.
Good idea @jaimergp ! Did you know how I could do that ? Could it be specific to Azur CI or does it have to work in all CI ?

Reply:
I guess that checking whether [`$CI`](https://github.com/conda-forge/staged-recipes/blob/main/.azure-pipelines/azure-pipelines-linux.yml#L19) is defined and not empty should be enough!

Reply:
Thanks, I did it : https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=444242&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=1745